core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -1417,29 +1416,45 @@ short ExExeUtilHBaseBulkLoadTcb::work()
               masterGlob->getStatement()->getContext()->setSqlParserFlags(0x20000);
             }
           }
+        ComDiagsArea *diagsArea = getDiagsArea();
 
         cliRC = cliInterface()->executeImmediate(loadQuery,
             NULL,
             NULL,
             TRUE,
-            &rowsAffected_);
-          if (parserFlagSet)
-            masterGlob->getStatement()->getContext()->resetSqlParserFlags(0x20000);
+            &rowsAffected_,
+            FALSE,
+            diagsArea);
 
+        if (parserFlagSet)
+            masterGlob->getStatement()->getContext()->resetSqlParserFlags(0x20000);
         if (cliRC < 0)
         {
           rowsAffected_ = 0;
           cliInterface()->retrieveSQLDiagnostics(getDiagsArea());
           step_ = LOAD_END_ERROR_;
           break;
         }
-        
-        step_ = COMPLETE_BULK_LOAD_;
+        else {
+           step_ = COMPLETE_BULK_LOAD_;
+           ComCondition *cond;
+           Lng32 entryNumber;
+           while ((cond = diagsArea->findCondition(EXE_ERROR_ROWS_FOUND, &entryNumber)) != NULL) {
+              errorRowCount = cond->getOptionalInteger(0);
+              diagsArea->deleteWarning(entryNumber);
+           }
+           // Need to clear the diags Area to get the correct rowsAffected for the LOAD COMPLETE command
+           // Hence, we might lose any other warnings created at the time of loading like error during
+           // logging error rows
+           diagsArea->clear();
+        }
         if (rowsAffected_ == 0)
           step_ = LOAD_END_;
 
-        sprintf(statusMsgBuf_,"       Rows Processed: %ld %c",rowsAffected_, '\n' );
+        sprintf(statusMsgBuf_,      "       Rows Processed: %ld %c",rowsAffected_+errorRowCount, '\n' );
```

Comment:
It will be good to make a documentation change explaining this. RowsProcessed is rows read or some kind of total. To get the actual number of rows loaded user should subtract the two numbers (or maybe it comes out as a summary of the statement?)